src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1635,6 +1636,21 @@ components:
       required:
         - reuse_uplod
         - reuse_group
+    Scancode:
+      type: object
+      properties:
+        license:
+          type: string
```

Comment:
Type should be boolean I believe.

Reply:
Also, please add `Scancode` to `ScanOptions` at https://github.com/fossology/fossology/blob/864570d6bd11f82083be80c314a5e1a4df061a87/src/www/ui/api/documentation/openapi.yaml#L1418

Reply:
And update the patch version in yaml file as well.